scripts/build-hotrod-image.sh

```diff
@@ -99,7 +99,10 @@ make build-${binary}
 bash scripts/build-upload-a-docker-image.sh -l -b -c "${binary}" -d cmd/"${binary}" -p "${current_platform}" -t release
 
 echo '::group:: docker compose'
-JAEGER_VERSION=$GITHUB_SHA REGISTRY="localhost:5000/" docker compose -f "$docker_compose_file" up -d
+JAEGER_VERSION=$GITHUB_SHA \
+  HOTROD_VERSION=$GITHUB_SHA \
```

Comment:
is this env var being used anywhere?

Reply:
in examples/hotrod/docker-compose-v2.yml

Reply:
@yurishkuro I don't see it [here](https://github.com/jaegertracing/jaeger/blob/main/examples/hotrod/docker-compose-v2.yml) -  are we planning to add it?

Reply:
it's in this PR https://github.com/jaegertracing/jaeger/pull/6206/files#diff-007552fc6e14179ca42b3d84e756efe9966277761f330792c385031b3017474fR20

Reply:
ah got it - sorry I missed that